allennlp/semparse/worlds/wikitables_variable_free_world.py

```diff
@@ -101,41 +106,46 @@ def _map_name(self, name: str, keep_mapping: bool = False) -> str:
         if name not in types.COMMON_NAME_MAPPING and name not in self.local_name_mapping:
             if not keep_mapping:
```

Comment:
I'd recommend moving out the "add name to mapping" functionality from this method.  Basically, just take this whole block under `if name not in types.COMMON_NAME_MAPPING ...` and move it into a separate function (removing the `keep_mapping` argument).  Then `_map_name` can just do the `else` block below.

Reply:
Made some changes in this method for the next PR. Will make a separate method there.